### PR TITLE
Inclusion of Qmin and Qmax and update test data

### LINKED ISSUE #61: Qmin Qmax to be within mask range

### PR DESCRIPTION
**Pull request by cadenmyers13**
closes #61. I did this in reference to this comment https://github.com/diffpy/diffpy.fourigui/pull/47#issuecomment-2465570287. I created the new data through the test function which might not be ideal but this was the quickest way I could do it. I had to create new fourier transformed data as well for the other test because it relied on the old data. Pytest passes on this now, and when I ran pytest on the old source code (`>=`/`<=` instead of `>`/`<`) it passed. Now that I'm thinking about it, I might be able to make new test data from the included test data files by just trimming the data...

TL;DR I changed the test data and pytest failed when expected and passed when expected. Also, the source code has been updated for the desired Q masking. 

@bobleesj mention you just so you can see

### CODE REVIEW

**Comment by bobleesj** [comment]
> @bobleesj mention you just so you can see

Thanks for tagging. Yes, I am "watching" this repository.

**Comment by sbillinge** [comment]
Thanks Caden. I need to take a quick look at this.  On my phone right now.

**Comment by sbillinge** [comment]
Closing.  replaced by #63
